### ISSUE TITLE
Update to v0.6.1?

### ISSUE THREAD

**Issue by FNAF-fan-9000** [BombasticTom/CustomizableLuaSettings#1]
The title says everything but I have a feeling that most of the options don't work because of the addition of things like week 7's cutscenes and stress' picospeaker.json file (and the hardcoding of the blammed lights event into only week 3/the philly stage)

**Comment by BombasticTom** [comment]
lmao this is so outdated there's a (MUCH) newer version on [gamebanana](https://gamebanana.com/mods/358754)

I haven't checked if it works with 0.6 tho

**Comment by FNAF-fan-9000** [comment]
I know, the latest version doesn't work for 0.6, it doesn't do anything, with the exception (from what I've tested) of the "Well well well" setting

**Comment by BombasticTom** [comment]
damn i'll have to look into that

**Comment by BombasticTom** [comment]
every option seems to be working except for choosing a character

**Comment by BombasticTom** [comment]
ohh wait have you used blammed lights?

**Comment by FNAF-fan-9000** [comment]
I mean, I was curious as to what it did, now that the blammed lights are hardcoded...

**Comment by BombasticTom** [comment]
someone told me that it doesn't work with 0.6.0 because it's broken
and they playtested with 0.6.1 and said it works

**Comment by FNAF-fan-9000** [comment]
huh, maybe it's because I tried a song that doesn't use the philly stage?

**Comment by FNAF-fan-9000** [comment]
nope, still doesn't work, is it because of the rework of how you set up the event?
